app/code/Magento/AdvancedSearch/Test/Unit/Model/Indexer/Fulltext/Plugin/CustomerGroupTest.php

```diff
@@ -3,52 +3,65 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
 
 namespace Magento\AdvancedSearch\Test\Unit\Model\Indexer\Fulltext\Plugin;
 
-use Magento\AdvancedSearch\Model\Indexer\Fulltext\Plugin\CustomerGroup;
+use Magento\AdvancedSearch\Model\Client\ClientOptionsInterface;
+use Magento\AdvancedSearch\Model\Indexer\Fulltext\Plugin\CustomerGroup as CustomerGroupPlugin;
+use Magento\CatalogSearch\Model\Indexer\Fulltext as FulltextIndexer;
+use Magento\Customer\Model\Group as CustomerGroupModel;
+use Magento\Customer\Model\ResourceModel\Group as CustomerGroupResourceModel;
+use Magento\Framework\Indexer\IndexerInterface;
+use Magento\Framework\Indexer\IndexerRegistry;
 use Magento\Framework\Search\EngineResolverInterface;
+use PHPUnit\Framework\TestCase;
 
-class CustomerGroupTest extends \PHPUnit\Framework\TestCase
+/**
+ * @covers Magento\AdvancedSearch\Model\Indexer\Fulltext\Plugin\CustomerGroup
+ */
+class CustomerGroupTest extends TestCase
 {
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject|\Magento\Framework\Indexer\IndexerInterface
+     * Testable Object
+     *
+     * @var CustomerGroupPlugin
+     */
+    protected $model;
+
+    /**
+     * @var IndexerInterface|MockObject
      */
     protected $indexerMock;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject|\Magento\Customer\Model\ResourceModel\Group
+     * @var Group|MockObject
      */
     protected $subjectMock;
```

Comment:
How do you like the idea to change scope of the properties for this test case to `private`?

Reply:
Indeed. Thank you for noticing!